app/policies/target_policy.rb

```diff
@@ -1,20 +1,5 @@
 class TargetPolicy < ApplicationPolicy
-  class Scope
-    def initialize(user, scope)
-      @user  = user
-      @scope = scope
-    end
-
-    def resolve
-      scope.all
-    end
-
-    private
-
-    attr_reader :user, :scope
-  end
-
   def create?
-    true
+    @user
```

Comment:
How this works? Can we add a test for this `create` method in spec/policies ?
I usually use `user.present?` here but maybe this works too